packages/@lwc/shared/src/void-elements.ts

```diff
@@ -11,7 +11,7 @@ import { HTML_NAMESPACE } from './namespaces';
 // e.g. `</tagName>` or `<tagName />`. For instance, `<meta>` closes on its own; no need for a slash.
 // These only come from HTML; there are no void elements in the SVG or MathML namespaces.
 // See: https://html.spec.whatwg.org/multipage/syntax.html#syntax-tags
-const VOID_ELEMENTS = /*@__PURE__*/ [
+const VOID_ELEMENTS = [
```

Comment:
Is this change supposed to be part of this PR?

Reply:
Yes sorry I forgot to explain, Rollup v4 errors on useless `PURE` annotations now.

> Rollup will now warn for @__PURE__ and @__NO_SIDE_EFFECTS__ annotations in invalid locations (https://github.com/rollup/rollup/pull/5165)